CRITICAL - 2023-12-05 11:44:43 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Model.php(151): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\OrgCrud.php(10): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:47:57 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Model.php(151): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\OrgCrud.php(10): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:51:07 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Model.php(151): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\OrgCrud.php(10): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:51:48 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Model.php(151): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\OrgCrud.php(10): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:51:49 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Model.php(151): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\OrgCrud.php(10): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:52:31 --> Undefined variable $users
in APPPATH\Views\organisations.php on line 28.
 1 APPPATH\Views\organisations.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $users', 'C:\\xampp\\htdocs\\cc\\app\\Views\\organisations.php', 28)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\cc\\app\\Views\\organisations.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('organisations', [], true)
 5 APPPATH\Controllers\OrgCrud.php(12): view('organisations')
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:52:46 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Model.php(151): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\OrgCrud.php(10): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:53:25 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Model.php(151): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\OrgCrud.php(10): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:53:26 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Model.php(151): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\OrgCrud.php(10): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:54:45 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Common.php(363): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\OrgCrud.php(10): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:54:46 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Common.php(363): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\OrgCrud.php(10): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:54:46 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 52.
 1 SYSTEMPATH\Database\Config.php(83): CodeIgniter\Database\Database->load([], 'default')
 2 SYSTEMPATH\Common.php(363): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\OrgCrud.php(10): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:55:24 --> Cannot use object of type PgSql\Connection as array
in APPPATH\Views\organisations.php on line 30.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('organisations', [], true)
 4 APPPATH\Controllers\OrgCrud.php(12): view('organisations', [...])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
ERROR - 2023-12-05 11:56:42 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;id&quot; does not exist
LINE 3: ORDER BY &quot;id&quot; DESC
                 ^ in C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 187)
#1 C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php(187): pg_query(Object(PgSql\Connection), 'SELECT *\nFROM "...')
#2 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 C:\xampp\htdocs\cc\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 C:\xampp\htdocs\cc\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\cc\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp\htdocs\cc\app\Controllers\OrgCrud.php(11): CodeIgniter\BaseModel->findAll()
#8 C:\xampp\htdocs\cc\system\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
#9 C:\xampp\htdocs\cc\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
#10 C:\xampp\htdocs\cc\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\cc\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\cc\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-12-05 11:56:42 --> Call to a member function getResult() on bool
in SYSTEMPATH\Model.php on line 238.
 1 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH\Controllers\OrgCrud.php(11): CodeIgniter\BaseModel->findAll()
 3 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
ERROR - 2023-12-05 11:57:09 --> ErrorException: pg_query(): Query failed: ERROR:  column &quot;id&quot; does not exist
LINE 3: ORDER BY &quot;id&quot; DESC
                 ^ in C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 187)
#1 C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php(187): pg_query(Object(PgSql\Connection), 'SELECT *\nFROM "...')
#2 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 C:\xampp\htdocs\cc\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 C:\xampp\htdocs\cc\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\cc\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp\htdocs\cc\app\Controllers\OrgCrud.php(11): CodeIgniter\BaseModel->findAll()
#8 C:\xampp\htdocs\cc\system\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
#9 C:\xampp\htdocs\cc\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
#10 C:\xampp\htdocs\cc\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\cc\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\cc\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-12-05 11:57:09 --> Call to a member function getResult() on bool
in SYSTEMPATH\Model.php on line 238.
 1 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 2 APPPATH\Controllers\OrgCrud.php(11): CodeIgniter\BaseModel->findAll()
 3 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:59:06 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\organisations.php on line 28.
 1 APPPATH\Views\organisations.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'C:\\xampp\\htdocs\\cc\\app\\Views\\organisations.php', 28)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\cc\\app\\Views\\organisations.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('organisations', [], true)
 5 APPPATH\Controllers\OrgCrud.php(12): view('organisations', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:59:33 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\organisations.php on line 28.
 1 APPPATH\Views\organisations.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'C:\\xampp\\htdocs\\cc\\app\\Views\\organisations.php', 28)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\cc\\app\\Views\\organisations.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('organisations', [], true)
 5 APPPATH\Controllers\OrgCrud.php(12): view('organisations', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 11:59:47 --> Undefined array key "id"
in APPPATH\Views\organisations.php on line 30.
 1 APPPATH\Views\organisations.php(30): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\cc\\app\\Views\\organisations.php', 30)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\cc\\app\\Views\\organisations.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('organisations', [], true)
 5 APPPATH\Controllers\OrgCrud.php(12): view('organisations', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 13:45:17 --> Undefined array key "id"
in APPPATH\Views\organisations.php on line 39.
 1 APPPATH\Views\organisations.php(39): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\cc\\app\\Views\\organisations.php', 39)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\cc\\app\\Views\\organisations.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('organisations', [], true)
 5 APPPATH\Controllers\OrgCrud.php(12): view('organisations', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 13:58:47 --> Undefined array key "id"
in APPPATH\Views\editorg.php on line 28.
 1 APPPATH\Views\editorg.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\cc\\app\\Views\\editorg.php', 28)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\cc\\app\\Views\\editorg.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('editorg', [], true)
 5 APPPATH\Controllers\OrgCrud.php(29): view('editorg', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->singleUser('12')
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 13:59:24 --> Updates are not allowed unless they contain a "where" or "like" clause.
in SYSTEMPATH\Model.php on line 380.
 1 SYSTEMPATH\BaseModel.php(949): CodeIgniter\Model->doUpdate(null, [...])
 2 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update(null, [...])
 3 APPPATH\Controllers\OrgCrud.php(37): CodeIgniter\Model->update(null, [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 14:03:08 --> Updates are not allowed unless they contain a "where" or "like" clause.
in SYSTEMPATH\Model.php on line 380.
 1 SYSTEMPATH\BaseModel.php(949): CodeIgniter\Model->doUpdate(null, [...])
 2 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update(null, [...])
 3 APPPATH\Controllers\OrgCrud.php(37): CodeIgniter\Model->update(null, [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 14:04:47 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1726.
 1 SYSTEMPATH\BaseModel.php(1726): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(916): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Controllers\OrgCrud.php(37): CodeIgniter\Model->update([...])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 14:04:59 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1726.
 1 SYSTEMPATH\BaseModel.php(1726): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(916): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Controllers\OrgCrud.php(37): CodeIgniter\Model->update([...])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 14:07:47 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1726.
 1 SYSTEMPATH\BaseModel.php(1726): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(916): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Controllers\OrgCrud.php(37): CodeIgniter\Model->update([...])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 14:10:10 --> Illegal offset type
in SYSTEMPATH\HTTP\RequestTrait.php on line 274.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(554): CodeIgniter\HTTP\Request->fetchGlobal('request', [...], 516, 0)
 2 APPPATH\Controllers\OrgCrud.php(36): CodeIgniter\HTTP\IncomingRequest->getVar([...])
 3 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 14:11:04 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1726.
 1 SYSTEMPATH\BaseModel.php(1726): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(916): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Controllers\OrgCrud.php(41): CodeIgniter\Model->update([...])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 14:12:09 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 930.
 1 SYSTEMPATH\BaseModel.php(930): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Controllers\OrgCrud.php(41): CodeIgniter\Model->update(null, [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 14:12:17 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 930.
 1 SYSTEMPATH\BaseModel.php(930): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Controllers\OrgCrud.php(41): CodeIgniter\Model->update(null, [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
CRITICAL - 2023-12-05 14:13:53 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 930.
 1 SYSTEMPATH\BaseModel.php(930): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\OrgCrud.php(41): CodeIgniter\Model->update('12', [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\cc\\public\\index.php')
ERROR - 2023-12-05 14:17:10 --> ErrorException: pg_query(): Query failed: ERROR:  column org.id does not exist
LINE 2: WHERE &quot;org&quot;.&quot;id&quot; IN ('12')
              ^ in C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 187)
#1 C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php(187): pg_query(Object(PgSql\Connection), 'UPDATE "org" SE...')
#2 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "org" SE...')
#3 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "org" SE...')
#4 C:\xampp\htdocs\cc\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE "org" SE...', Array, false)
#5 C:\xampp\htdocs\cc\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#6 C:\xampp\htdocs\cc\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#7 C:\xampp\htdocs\cc\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#8 C:\xampp\htdocs\cc\app\Controllers\OrgCrud.php(41): CodeIgniter\Model->update('12', Array)
#9 C:\xampp\htdocs\cc\system\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
#10 C:\xampp\htdocs\cc\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
#11 C:\xampp\htdocs\cc\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\cc\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\cc\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2023-12-05 14:17:16 --> ErrorException: pg_query(): Query failed: ERROR:  column org.id does not exist
LINE 2: WHERE &quot;org&quot;.&quot;id&quot; IN ('12')
              ^ in C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 187)
#1 C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php(187): pg_query(Object(PgSql\Connection), 'UPDATE "org" SE...')
#2 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "org" SE...')
#3 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "org" SE...')
#4 C:\xampp\htdocs\cc\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE "org" SE...', Array, false)
#5 C:\xampp\htdocs\cc\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#6 C:\xampp\htdocs\cc\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#7 C:\xampp\htdocs\cc\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#8 C:\xampp\htdocs\cc\app\Controllers\OrgCrud.php(41): CodeIgniter\Model->update('12', Array)
#9 C:\xampp\htdocs\cc\system\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
#10 C:\xampp\htdocs\cc\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
#11 C:\xampp\htdocs\cc\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\cc\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\cc\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2023-12-05 14:17:49 --> ErrorException: pg_query(): Query failed: ERROR:  column org.id does not exist
LINE 3: AND &quot;org&quot;.&quot;id&quot; IN ('12')
            ^ in C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 187)
#1 C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php(187): pg_query(Object(PgSql\Connection), 'UPDATE "org" SE...')
#2 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "org" SE...')
#3 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "org" SE...')
#4 C:\xampp\htdocs\cc\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE "org" SE...', Array, false)
#5 C:\xampp\htdocs\cc\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#6 C:\xampp\htdocs\cc\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#7 C:\xampp\htdocs\cc\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#8 C:\xampp\htdocs\cc\app\Controllers\OrgCrud.php(41): CodeIgniter\Model->update('12', Array)
#9 C:\xampp\htdocs\cc\system\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
#10 C:\xampp\htdocs\cc\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
#11 C:\xampp\htdocs\cc\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\cc\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\cc\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2023-12-05 14:17:57 --> ErrorException: pg_query(): Query failed: ERROR:  column org.id does not exist
LINE 3: AND &quot;org&quot;.&quot;id&quot; IN ('5')
            ^ in C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 187)
#1 C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php(187): pg_query(Object(PgSql\Connection), 'UPDATE "org" SE...')
#2 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "org" SE...')
#3 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "org" SE...')
#4 C:\xampp\htdocs\cc\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE "org" SE...', Array, false)
#5 C:\xampp\htdocs\cc\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#6 C:\xampp\htdocs\cc\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#7 C:\xampp\htdocs\cc\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#8 C:\xampp\htdocs\cc\app\Controllers\OrgCrud.php(41): CodeIgniter\Model->update('5', Array)
#9 C:\xampp\htdocs\cc\system\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
#10 C:\xampp\htdocs\cc\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
#11 C:\xampp\htdocs\cc\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\cc\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\cc\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2023-12-05 14:18:48 --> ErrorException: pg_query(): Query failed: ERROR:  column org.id does not exist
LINE 2: WHERE &quot;org&quot;.&quot;id&quot; IN ('12')
              ^ in C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 187)
#1 C:\xampp\htdocs\cc\system\Database\Postgre\Connection.php(187): pg_query(Object(PgSql\Connection), 'UPDATE "org" SE...')
#2 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(693): CodeIgniter\Database\Postgre\Connection->execute('UPDATE "org" SE...')
#3 C:\xampp\htdocs\cc\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE "org" SE...')
#4 C:\xampp\htdocs\cc\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE "org" SE...', Array, false)
#5 C:\xampp\htdocs\cc\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#6 C:\xampp\htdocs\cc\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#7 C:\xampp\htdocs\cc\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#8 C:\xampp\htdocs\cc\app\Controllers\OrgCrud.php(39): CodeIgniter\Model->update('12', Array)
#9 C:\xampp\htdocs\cc\system\CodeIgniter.php(942): App\Controllers\OrgCrud->update()
#10 C:\xampp\htdocs\cc\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\OrgCrud))
#11 C:\xampp\htdocs\cc\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\cc\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\cc\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
